docs/rum_mobile_vitals.md

```diff
@@ -0,0 +1,41 @@
+
+![Event Timings and Mobile Vitals][4]
+
+Mobile vitals include recommended benchmark ranges that correlate directly to your application's user experience. You can see where a metric scores on the range and click **Search Views With Poor Performance** to apply a filter in your search query.
+
+## Metrics
+
+The following metrics provide insight into your mobile application's performance.
+
+| Measurement                    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Slow renders                   | To ensure a smooth, [jank-free][5] user experience, your application should render frames in under 60 seconds. <br /><br />  RUM tracks the application’s [display refresh rate][6] using `@view.refresh_rate_average` and `@view.refresh_rate_min` view attributes. <br /><br />  With slow rendering, you can monitor which views are taking longer than 40, 50, or 60 seconds to render. |
+| Frozen frames                  | Frames that take longer than 700ms to render appear as stuck and unresponsive in your application. These are classified as [frozen frames][7]. <br /><br />  RUM tracks `long task` events with the duration for any task taking longer then 100ms to complete. <br /><br />  With frozen frames, you can monitor which views appear frozen (taking longer than 700ms to render) to your end users and eliminate jank in your application.                                                                                                                                                                                                 |
```

Comment:
Shall we add a note here that customer can set a custom threshold to replace the default `100ms`? If so they can use the following line in the configuration:
```
val configuration = Configuration.Builder(…)
        // …
        .trackLongTasks(250L) // track tasks longer than 250ms as long tasks
        .build()
```

Reply:
```suggestion
| Frozen frames                  | Frames that take longer than 700ms to render appear as stuck and unresponsive in your application. These are classified as [frozen frames][7]. <br /><br />  RUM tracks `long task` events with the duration for any task taking longer then 100ms to complete. <br /><br />  With frozen frames, you can monitor which views appear frozen (taking longer than 700ms to render) to your end users and eliminate jank in your application. <br /> **Note:** To replace the default `100 ms` duration, set a custom threshold in your configuration: <br /> <br><br>val configuration = Configuration.Builder(…)<br>        // …<br>        .trackLongTasks(250L) // track tasks longer than 250ms as long tasks<br>        .build()                         |
```

Reply:
I think this is confusing to add the configuration snippet about changing long task threshold to frozen frames definition - since it might mislead the reader into thinking that they need to override this default value. I say we add this snippet in the long task definition (data_collected) page instead.

Reply:
Gotcha, what if we add this example in the **Automatically track long tasks** section of [RUM Android Advanced Configuration](https://docs.datadoghq.com/real_user_monitoring/android/advanced_configuration/?tab=kotlin)?

Reply:
Good idea!